src/components/SearchResults.js

```diff
@@ -18,12 +18,7 @@ const SearchResults = prop => {
   };
 
   const handleClick = field => {
-    if (
-      field === "firstName" ||
-      field === "title" ||
-      field === "surname" ||
-      field === "email"
-    ) {
+    if (field === "firstName" || field === "title" || field === "surname") {
```

Comment:
You could leave in your email field, it's just your map function below on line 22 which converts to lowercase isn't being used anywhere (you aren't currently saving it to a variable to use later on.) In your sort function, you are sorting values which are a mixture of lowercase & uppercase, this is fine for most columns, but your email address column has a mixture of upper & lowercase values so it is not sorting correctly. Send me a message on slack if you need me to explain more :)